INSTRUCTION.md

```diff
@@ -0,0 +1,63 @@
+# INSTRUCTION.md
+
+## 1. Як запускати контейнер MySQL
+
+Щоб запустити контейнер MySQL, виконайте наступну команду в терміналі:
+
+docker run -d --name mysql-container \
+  -e MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD} \
+  -e MYSQL_DATABASE=${MYSQL_DATABASE} \
+  -e MYSQL_USER=${MYSQL_USER} \
+  -e MYSQL_PASSWORD=${MYSQL_PASSWORD} \
+  -v mysql_data:/var/lib/mysql -p 3306:3306 mysql-local:1.0.0
+
+# Ця команда:
+
+Створює контейнер з ім'ям mysql-container.
+Встановлює пароль для користувача root як rootpassword.
+Створює базу даних app_db та користувача app_user з паролем 1234.
```

Comment:
Avoid hardcoding sensitive information such as passwords in the instructions. Instead, use placeholders or environment variables to ensure security.